eo-parser/src/test/java/org/eolang/parser/XMIRTest.java

```diff
@@ -144,6 +130,20 @@ void convertsAntlrToEbnf() throws Exception {
                 Matchers.containsString("<text> := 'DQ' 'DQ' 'DQ' [")
             )
         );
+        Files.write(
+            temp.resolve("article.tex"),
+            new TextOf(new ResourceOf("tex/ebnf.tex"))
+                .asString()
+                .replace("EBNF", ebnf)
+                .getBytes(StandardCharsets.UTF_8)
+        );
+        new Jaxec(
```

Comment:
@yegor256 What do you think if we test this in a separate unit test? It looks like a clear example of `Cuckoo` anti-pattern:
https://www.yegor256.com/2018/12/11/unit-testing-anti-patterns.html